examples/example.js

```diff
@@ -1,140 +1,205 @@
-const { Wallet } = require('ethers')
-const grpc = require('@grpc/grpc-js')
-const protoLoader = require('@grpc/proto-loader')
-const { ethers } = require("ethers")
+const _ = require("lodash");
+const grpc = require("@grpc/grpc-js");
+const protoLoader = require("@grpc/proto-loader");
+const { ethers } = require("ethers");
+const { Wallet } = ethers;
 
-const {
-  TaskType, TriggerType
-} = require('./static_codegen/avs_pb')
+const { TaskType, TriggerType } = require("./static_codegen/avs_pb");
 
 // Load the protobuf definition
-const packageDefinition = protoLoader.loadSync('../protobuf/avs.proto', {
-    keepCase: true,
-    longs: String,
-    enums: String,
-    defaults: true,
-    oneofs: true
-})
+const packageDefinition = protoLoader.loadSync("../protobuf/avs.proto", {
+  keepCase: true,
+  longs: String,
+  enums: String,
+  defaults: true,
+  oneofs: true,
+});
 
-const env = process.env.env || "development"
-const privateKey = process.env.PRIVATE_KEY // Make sure to provide your private key with or without the '0x' prefix
+const env = process.env.ENV || "development";
+const privateKey = process.env.PRIVATE_KEY; // Make sure to provide your private key with or without the '0x' prefix
 
 const config = {
   development: {
-    AP_AVS_RPC: 'localhost:2206',
+    AP_AVS_RPC: "localhost:2206",
     TEST_TRANSFER_TOKEN: "0x2e8bdb63d09ef989a0018eeb1c47ef84e3e61f7b",
-    TEST_TRANSFER_TO: '0xe0f7D11FD714674722d325Cd86062A5F1882E13a',
-    ORACLE_PRICE_CONTRACT: '0x694AA1769357215DE4FAC081bf1f309aDC325306',
+    TEST_TRANSFER_TO: "0xe0f7D11FD714674722d325Cd86062A5F1882E13a",
+    ORACLE_PRICE_CONTRACT: "0x694AA1769357215DE4FAC081bf1f309aDC325306",
   },
 
   staging: {
-    AP_AVS_RPC: 'aggregator-holesky.avaprotocol.org:2206',
+    AP_AVS_RPC: "aggregator-holesky.avaprotocol.org:2206",
     TEST_TRANSFER_TOKEN: "0x2e8bdb63d09ef989a0018eeb1c47ef84e3e61f7b",
-    TEST_TRANSFER_TO: '0xe0f7D11FD714674722d325Cd86062A5F1882E13a',
-    ORACLE_PRICE_CONTRACT: '0x694AA1769357215DE4FAC081bf1f309aDC325306',
+    TEST_TRANSFER_TO: "0xe0f7D11FD714674722d325Cd86062A5F1882E13a",
+    ORACLE_PRICE_CONTRACT: "0x694AA1769357215DE4FAC081bf1f309aDC325306",
+    RPC_PROVIDER: "https://rpc.sepolia.avaprotocol.org",
+  },
+
+  minato: {
+    AP_AVS_RPC: "aggregator-minato.avaprotocol.org:2306",
+    TEST_TRANSFER_TOKEN: "0xBA33747043d09868946978Dd935130490a083458",
+    TEST_TRANSFER_TO: "0xe0f7D11FD714674722d325Cd86062A5F1882E13a",
+    ORACLE_PRICE_CONTRACT: "0x0ee7f0f7796Bd98c0E68107c42b21F5B7C13bcA9",
+    RPC_PROVIDER: "https://rpc.minato.soneium.org",
   },
```

Comment:
Environmental Variables:
PRIVATE_KEY=<private_key>
ENV=minato

Reply:
minato variables are added, and a RPC_PROVIDER is added to get token balances and read on-chain transactions.